extras/ci/platformio.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -eux
 
-pip install --user platformio
+pip3 install --user platformio
```

Comment:
You should revert this change; I fixed the CI in https://github.com/bblanchon/ArduinoJson/commit/3975b073716f629d010c24d89b1d742e2f9bc6f9